src/guides/v2.3/extension-dev-guide/message-queues/config-mq.md

```diff
@@ -13,7 +13,6 @@ The message queue topology is a {{site.data.var.ce}} feature. It can be included
 
 Configuring the message queue topology involves creating and modifying the following configuration files in the `<module>/etc` directory:
 
-*  [`queue.xml`](#queuexml) - Defines brokers that processes topics. Use for `db` ([MySQL]) connections only. Do not create this file for `amqp` ([RabbitMQ]) connections.
```

Comment:
I guess it would be also nice to add a Notice that `queue.xml` is deprecated since 2.2 version (If I'm not mistaken) and the separate files queue_consumer, queue_topology, queue_publisher should be used instead, etc.
Just in case someone is bumped into `queue.xml` file it in the code and looking for its purpose. :)

Reply:
Hi @dmytro-ch , not really. Officially its deprecated and its true, but there were a bug in 2.3.x versions which still required queue.xml for DB communication.
Its was removed if I'm not mistaken was 2.3.4. 

So I'm not really sure we need deprecation notice, as whole structure changed. 
Also in bottom of the page you can see there are a link for migration.

But up to you :-)

Reply:
Hi @dmytro-ch , not really. Officially its deprecated and its true, but there were a bug in 2.3.x versions which still required queue.xml for DB communication.
Its was removed if I'm not mistaken was 2.3.4. 

So I'm not really sure we need deprecation notice, as whole structure changed. 
Also in bottom of the page you can see there are a link for migration.

But up to you :-)

Reply:
@nuzil, you are right. I didn't notice that link.
Thank you for the details!